ansible/provision-tiered-storage-cluster.yml

```diff
@@ -0,0 +1,32 @@
+# Creates a TLS enabled cluster with tiered storage local CA in artifacts/ca for testing purposes
```

Comment:
minor:

Think this is missing a word or two. maybe this?

"with tiered storage, using a local CA [...]"